### ISSUE TITLE
add Publications detail view

### ISSUE THREAD

**Issue by jmcmichael** [genome/civic-client#491]
include publication summary w/ all details, plus evidence grid of items that reference the publication.

**Comment by acoffman** [comment]
Now that we have PMCIDs as a proxy for "open access" we should indicate that status and link out on the publication detail view.

**Comment by acoffman** [comment]
Doing some cleaning up of the issues and found some details of what we want to display on this view from an older server side issue. Closing that one as a duplicate and moving that content here.

From malachi:



It would be great to a have a view in the interface that showed the current 'bibliography' of CIViC. This would be the list of sources we have obtained 1 to n evidence statements from.

Use cases:
*  We give credit to the authors of papers, while also giving a sense of the number and diversity of papers CIViC leans on.
* User can see if their favorite paper is already in CIViC
* User can add a paper to the queue that they believe has value but hasn't been curated yet.

View could show:

* Compact citation pulled from PubMed as we display elsewhere
* Number of evidence statements pulled from the paper so far
* Status (Curated, pending, etc.)

This view could be a third view:
'Browse by Variant', 'Browse by Gene', and now 'Browse by Source'.
